src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX = 20;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        Names[] names = Names.values();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = colorSupplier.getRandomColor();
+        Figure figure = null;
```

Comment:
let's remove this variable and return a concrete figure in each case of switch case block)